File: python/x.py
num = [1,2,3,4,5,6,7,8,9,10]
l1 = []
l2 = []
for i in num:
    if i == 1:
        l1.append(i)
    else:
        for j in range(1,i-1):

            if i%num[j] == 0:
                l1.append(i)
                break
        else:
            l2.append(i)

print(l1)
print(l2)

File: others/x.cpp
#include <iostream>
using namespace std;

void display(void (*p)(int num))
{
    for (int i = 1; i <= 5; i++)
    {
        p(i);
    }
}
void print_numbers(int num)
{
    cout << num;
}
int main()
{
    // void (*p)(int); // void function pointer declaration
    printf("The values are :");
    display(print_numbers);
    return 0;
}
